packages/toggle/src/Toggle.tsx

```diff
@@ -417,6 +417,7 @@ function Toggle({
     >
       <button
         role="switch"
+        type="button"
```

Comment:
Didn't this change already get merged in a different PR?

Reply:
I... don't know where this change came from